### ISSUE TITLE
Undefined event name in webhook handler

### ISSUE THREAD

**Issue by iamogbz** [probot/serverless-lambda#12]
Just started using probot and while debugging noticed this:
```
03:01:34.721Z DEBUG probot: Webhook received
event: {
  "event": "undefined.synchronize",
  "installation": 101010,
  "repository": "owner/repo"
}
```
Traced it to
https://github.com/probot/serverless-lambda/blob/ee2e66125b930552d647e12676ef98b19fe10688/index.js#L45

Github might have changed the headers sent because at some point my bot was working:
```
X-Github-Event: pull_request
```
Current header has a slight spelling difference.

~EDIT: GitHub seems to have fixed the header issue in their request but making this more robust would still be a good idea.~

**Comment by mrchief** [comment]
In my case the header is `X-GitHub-Event` but I still get `undefined.XXX`. Not sure why this is happening.

**Comment by JasonEtco** [comment]
Can y'all see what `event.headers` is? Does it contain any headers at all?

**Comment by mrchief** [comment]
This is what I see in Cloudwatch logs

```
"event": {
        "event": "undefined.edited",
        "repository": "mrchief/githubActionsPoC"
    },
```

**Comment by iamogbz** [comment]
@JasonEtco it does contain headers. It's been a while but on the GitHub bot manage page the header sometimes gets displayed as `X-GitHub-Event` but when logging the raw `event.headers` received `X-Github-Event` is shown. I could consistently reproduce that at some point.

**Comment by aweiher** [comment]
Got `'X-Github-Event': 'push'` on GHE 2.16.4

**Comment by tcbyrd** [comment]
> `X-Github-Event`

Part of me just wants to figure out where the lowercase `h` is coming from and fix that, but This #14 is probably the more robust path forward 😬

Will get that tested and pushed to npm. Thanks for the discussion and help fixing it everyone!

**Comment by mrchief** [comment]
Http header names are case insensitive per the spec, so #14 is the way to go IMO
